x-pack/plugins/security_solution/server/lib/timeline/saved_object/timelines/index.ts

```diff
@@ -132,6 +90,44 @@ export const getTimelineOrNull = async (
   return timeline;
 };
 
+export const resolveTimeline = async (
+  request: FrameworkRequest,
+  timelineId: string,
+  timelineType: TimelineTypeLiteralWithNull = TimelineType.default
+): Promise<ResolvedTimelineSavedObject> => {
+  let timelineIdToUse = timelineId;
+  try {
+    if (timelineType === TimelineType.template) {
+      // TODO: When would this happen? The templateTimelineId wouldn't be defined in this scenario I thought?
```

Comment:
Yeah I think you're right. We can probably remove the entire `if` block. Seems like for the `getTimeline` the caller always checks to see if the type is a template and calls the template function instead.

Reply:
Yea, the only thing that throws me is that the `getTimeline` call uses the template timeline id, and this check looks at the type... @XavierM is it possible to have a timeline with type template, but not have a `template_timeline_id`?